x-pack/plugins/actions/server/usage/actions_telemetry.test.ts

```diff
@@ -754,6 +784,16 @@ Object {
         __slack: 7,
       },
       countTotal: 120,
+      countRunOutcomeByConnectorType: {
```

Comment:
I believe the paradigm we follow in other telemetry objects is to have the rule type id or connector type id last. That way, we could search by `countRunOutcomeByConnectorType.failed.*` and get all the different connector types that have failed, instead of having to look inside each connector type object. WDYT of switching this around to be similar?

Reply:
But the request is `We'd like to know which connector types are failing the most relative to their successful runs.`
Wouldn't it be more difficult to get a connector's success/failure ratio?

Reply:
Hmm...I'm not sure either of these options will make calculating the ratio per rule type easier :). We can leave it as is.

Reply:
my idea was getting it like: 
`count_connector_types_by_action_run_outcome_per_day.__slack.failure /
count_connector_types_by_action_run_outcome_per_day.__slack.success`

but 

`count_connector_types_by_action_run_outcome_per_day.failure.__slack./
count_connector_types_by_action_run_outcome_per_day.success.__slack`

would also do the same thing... IDK i can change it :)